frontend/src/components/content-types/ContentTypeForm.tsx

```diff
@@ -33,20 +38,34 @@ export const ContentTypeForm: FC<ComponentFormProps<IContentType>> = ({
   const { toast } = useToast();
   const { t } = useTranslate();
   const {
-    reset,
     control,
     register,
     setValue,
     formState: { errors },
     handleSubmit,
-  } = useForm<Partial<IContentType>>({
-    defaultValues: {
-      name: contentType?.name || "",
-      fields: contentType?.fields || FIELDS_FORM_DEFAULT_VALUES,
+  } = useForm<IContentType>({
+    defaultValues: contentType || {
+      name: "",
+      fields: FIELDS_FORM_DEFAULT_VALUES,
```

Comment:
I think contentType may contain additional attributes that are not relevant to the form (example : createdAt, updatedAt, ...)